etcd-manager/cmd/etcd-manager/main.go

```diff
@@ -100,6 +100,8 @@ func main() {
 	var volumeTags stringSliceFlag
 	flag.Var(&volumeTags, "volume-tag", "tag which volume is required to have")
 
+	flag.StringVar(&o.NetworkCIDR, "network-cidr", o.NetworkCIDR, "filtering for IP addresses by defining a comma separated list of CIDRs.")
```

Comment:
Please validate that this is only set for OpenStack, we want to make sure other providers don't expect it to work by mistake.

```suggestion
	flag.StringVar(&o.NetworkCIDR, "network-cidr", o.NetworkCIDR, "filtering for IP addresses by network CIDR (OpenStack only)")
```

Reply:
https://github.com/kubernetes-sigs/etcdadm/pull/317#discussion_r1122874600